test/integration/integration_test.exs

```diff
@@ -49,13 +49,21 @@ defmodule KafkaEx.Integration.Test do
     assert consumer_group_update_interval == 30000
   end
 
-  test "create_worker provides a default consumer_group of 'kafka_ex'" do
```

Comment:
OK, let me know what you want at a high level and I'll take a stab at implementing it.  How do you think the config.exs should be laid out so that the name of the consumer group and whether or not consumer groups are enabled is clear to the user?